typhos/display.py

```diff
@@ -74,6 +90,37 @@ def normalize_display_type(display_type):
         raise ValueError(f'Unrecognized display type: {display_type}') from ex
 
 
+def normalize_scroll_option(
+    scroll_option: Union[ScrollOptions, str, int]
+) -> ScrollOptions:
+    """
+    Normalize a given scroll option.
+
+    Parameters
+    ----------
+    display_type : ScrollOptions, str, or int
+        The display type.
+
+    Returns
+    -------
+    display_type : ScrollOptions
+        The normalized :class:`ScrollOptions`.
+
+    Raises
+    ------
+    ValueError
+        If the input cannot be made a :class:`ScrollOptions`.
+    """
+    try:
+        return ScrollOptions(scroll_option)
+    except Exception as ex:
+        if scroll_option in ScrollOptions.names:
```

Comment:
I think a shorter way might be (1) try `ScrollOptions(scroll_option)` (2) try `ScrollOptions[scroll_option]` (note brackets)

Reply:
You are correct, this is a copy/paste/nothink from a similar function above for the display types. I'll update both of them.